### PR TITLE
Bump to selenium 3.5.2

### PR DESCRIPTION
**Pull request by mach6**
- [x] By placing an `X` in the preceding checkbox, I verify that I am a PayPal employee or
I have signed the [Contributor License Agreement](https://github.com/paypal/SeLion#contributing)

---

### CODE REVIEW

**Comment by mach6** [comment]
@sebady @sundaramrajendran please review

File: client/src/main/java/com/paypal/selion/platform/grid/browsercapabilities/DefaultCapabilitiesBuilder.java [hunk 1]
```diff
@@ -49,9 +49,14 @@ public DesiredCapabilities getDefaultCapabilities() {
...
-        // refer here : http://code.google.com/p/selenium/wiki/DesiredCapabilities#Read-write_capabilities
+        // refer here: http://code.google.com/p/selenium/wiki/DesiredCapabilities#Read-write_capabilities
         // for understanding the relevance of this capability.
-        capability.setCapability(CapabilityType.UNEXPECTED_ALERT_BEHAVIOUR, "ignore");
+        // marionette does not like this capability. So, let's not set it in that
+        // instance.
+        if (!Grid.getWebTestSession().getBrowser().contains("firefox")
+                && getLocalConfigProperty(ConfigProperty.SELENIUM_USE_GECKODRIVER).equals("true")) {
+            capability.setCapability(CapabilityType.UNEXPECTED_ALERT_BEHAVIOUR, "ignore");
+        }
```
Comment (sebady) [inline client/src/main/java/com/paypal/selion/platform/grid/browsercapabilities/DefaultCapabilitiesBuilder.java]: Would it be better to have the `FireFoxCapabilitiesBuilder` do this?

**Review by sebady — APPROVED** [review]

File: client/src/main/java/com/paypal/selion/platform/grid/browsercapabilities/DefaultCapabilitiesBuilder.java (hunk 1 above)
Comment (mach6) [reply client/src/main/java/com/paypal/selion/platform/grid/browsercapabilities/DefaultCapabilitiesBuilder.java]: ideally, yes. However, this method in `DefaultCapabilitiesBuilder` happens after `FireFoxCapabilitiesBuilder`.  I'll see if I can devise a more elegant approach

**Comment by kevinx701** [comment]
where can I get SeLion 2.0.0 snapshot?

**Comment by mach6** [comment]
See the example `<repositories>` pom on this page - http://paypal.github.io/SeLion/html/documentation.html#getting-started

**Pull request merged**